core-blocks/spacer/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress
+ */
+import { __ } from '@wordpress/i18n';
+import ResizableBox from 're-resizable';
+import { InspectorControls } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+export const name = 'core/spacer';
+
+export const settings = {
+	title: __( 'Spacer' ),
+
+	description: __( 'Add an element with empty space and custom height.' ),
+
+	icon: 'image-flip-vertical',
+
+	category: 'layout',
+
+	attributes: {
+		height: {
+			type: 'number',
+			default: 100,
+		},
+	},
+
+	edit( { attributes, setAttributes, isSelected, toggleSelection } ) {
+		const { height } = attributes;
+
+		return [
```

Comment:
I refactored all existing blocks to use `<Fragment />` component instead of `[]` to avoid using `key` attributes and to be consistent. Can we update this block, too?

What do you think about having `edit` method in its own file? It was raised by the mobile team, as they experiment with replacing `edit` method with RN alternatives. See #5816 for reference. It would help them immensely to iterate before we come up with a set of primitives and other restrictions (if ever).

Reply:
For "edit" we have to sort out block server registration. It could be a possibility.